.github/workflows/linux.yml

```diff
@@ -25,7 +26,7 @@ jobs:
             build-type: release
 
           - os: ubuntu-20.04
-            container: ubuntu:22.10
+            container: ubuntu:23.04
```

Comment:
I don't fully understand this. What's the difference between os and container in this case? Why are we still setting os to ubuntu-20.04 while the container is 23.04?